README.md

```diff
@@ -26,27 +26,53 @@
 
 **[Rushema-Portfolio]** is the first page of my portfolio
 
-## 🛠 Built With <a name="built-with"></a>
+# 🛠 Built With <a name="built-with">Technologies</a>
 
-### Tech Stack <a name="tech-stack"></a>
+- HTML
+- CSS
+- Javascript
 
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md">HTML</a></li>
-    <li><a href="https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md">CSS</a></li>
-  </ul>
-</details>
+### Tech Stack <a name="tech-stack">HTML, CSS and JavaScript</a>
+
+The entire project was built with only HTML5, CSS3 and JavaScript.
+
+
+# 📖 [Online link porfolio] <a name="about-project"></a>
```

Comment:
[ OPTIONAL] 
- If you want to put the Live demo link in your README, make sure you always deply the recent changes because at the moment, the Mobile Menu changes are deployed to the live demo version.

Reply:
Thanks, Next time will change it